environments/eck-ror/start.sh

```diff
@@ -90,12 +90,38 @@ if [[ -z $ES_VERSION || -z $KBN_VERSION ]]; then
   show_help
 fi
 
+DOCKERFILE_DIR="./images/ubuntu"
+IMAGE_NAME="node-apm-app"
+TAG="latest"
+
+docker build -t "$IMAGE_NAME:$TAG" "$DOCKERFILE_DIR"
+
+if [ $? -eq 0 ]; then
+    echo "Docker image built successfully: $IMAGE_NAME:$TAG"
+else
+    echo "Docker image build failed."
+    exit 1
+fi
+
+
 echo "CONFIGURING K8S CLUSTER ..."
 kind create cluster --name ror-eck --config kind-cluster/kind-cluster-config.yml
 docker exec ror-eck-control-plane /bin/bash -c "sysctl -w vm.max_map_count=262144"
 docker exec ror-eck-worker        /bin/bash -c "sysctl -w vm.max_map_count=262144"
 docker exec ror-eck-worker2       /bin/bash -c "sysctl -w vm.max_map_count=262144"
 
+ # Load the Docker image into the Kind cluster
+    CLUSTER_NAME="ror-eck"
```

Comment:
1. indentation - formatting
2. WDYT about moving the image building here? (to be more precise after `trap cleanup EXIT` ... we will have building and loading in one place)
3. the same comment about explicit exit code handling